utils/ui.js

```diff
@@ -24,10 +26,10 @@ export function prettifyFloat(num, decimalPlaces = 2) {
     : num;
 }
 
-export function cleanSymbol(s) {
-  if (s === 'MDAI') return 'DAI';
-  if (s === 'DAI') return 'SAI';
-  return s;
+export function cleanSymbol(currency) {
```

Comment:
The need for this sort of utility function is a code smell -- it makes me want to take a step back and look at the overall situation with regard to SAI, DAI, MDAI. The way I've tried to address it in this PR is to explicitly wrap most values in the SAI and DAI classes that are defined in this codebase (in maker.js). But ideally we would go back into dai.js and the migration plugin and straighten things out there so that we can always use the classes defined in dai.js.